packages/view/src/ide/FakeIDEAdapter.ts

```diff
@@ -1,64 +1,65 @@
 ﻿import { injectable } from "tsyringe";
 
-import type { ClusterNode, EngineCommand, EngineCommandNames } from "types";
-import type { EngineMessage, EngineMessageEvent } from "types/EngineMessage";
+import type { ClusterNode, IDEMessage, IDEMessageEvent } from "types";
+import type { IDESentEvents } from "types/IDESentEvents";
 
 import fakeData from "../fake-assets/cluster-nodes.json";
 
 import type IDEPort from "./IDEPort";
 
 @injectable()
 export default class FakeIDEAdapter implements IDEPort {
-  public addAllEventListener(fetchAnalyzedData: (analyzedData: ClusterNode[]) => void) {
-    const onReceiveMessage = (e: EngineMessageEvent): void => {
-      const response = e.data;
-      if (response.command === "fetchAnalyzedData") {
-        const fetchedData = response.payload as unknown as ClusterNode[];
-        fetchAnalyzedData(fetchedData);
+  public addIDESentEventListener(events: IDESentEvents) {
+    const onReceiveMessage = (e: IDEMessageEvent): void => {
+      const responseMessage = e.data;
+      if (responseMessage.command === "fetchAnalyzedData") {
+        const fetchedData = responseMessage.payload as unknown as ClusterNode[];
+        events.fetchAnalyzedData(fetchedData);
       }
     };
 
     window.addEventListener("message", onReceiveMessage);
   }
 
-  public sendFetchAnalyzedDataCommand() {
-    const command: EngineCommand = {
+  public sendFetchAnalyzedDataMessage() {
+    const message: IDEMessage = {
       command: "fetchAnalyzedData",
     };
     setTimeout(() => {
-      this.executeCommand(command);
+      this.sendMessageToMe(message);
     }, 3000);
   }
 
   public setPrimaryColor(color: string) {
     sessionStorage.setItem("PRIMARY_COLOR", color);
-    const command: EngineCommand = {
+    const message: IDEMessage = {
       command: "updatePrimaryColor",
     };
-    this.executeCommand(command);
+    this.sendMessageToMe(message);
   }
 
-  private getResponseMessage(commandName: EngineCommandNames) {
-    switch (commandName) {
+  private convertMessage(message: IDEMessage) {
+    const { command } = message;
+
+    switch (command) {
       case "fetchAnalyzedData":
         return {
-          command: commandName,
+          command: command,
```

Comment:
```suggestion
          command,
```
destructuring assignment를 사용할 수 있을것 같습니다!

Reply:
오 좋습니다!!! 👍👍👍👍
담번 PR 에 반영하겠습니다.
(아니면 위 alias 반영하시면서 같이 작업 해주셔도 좋을 것 같습니다 ^^)

Reply:
네 이것도 같이 수정해보겠습니다 ㅎㅎ